eZ/Publish/Core/REST/Server/Tests/Output/ValueObjectVisitor/ExceptionTest.php

```diff
@@ -186,6 +219,51 @@ protected function getException()
      */
     protected function internalGetVisitor()
     {
-        return new ValueObjectVisitor\Exception();
+        return new ExceptionValueObjectVisitor(true, $this->getTranslatorMock());
+    }
+
+    /**
+     * Gets the exception visitor.
+     *
+     * @return \eZ\Publish\Core\REST\Server\Output\ValueObjectVisitor\Exception
+     */
+    protected function internalGetNonDebugVisitor(): ExceptionValueObjectVisitor
+    {
+        return new ExceptionValueObjectVisitor(false, $this->getTranslatorMock());
+    }
+
+    protected function getTranslatorMock(): TranslatorInterface
+    {
+        if (!isset($this->translatorMock)) {
+            $this->translatorMock = $this->getMockBuilder(TranslatorInterface::class)
+                 ->disableOriginalConstructor()
+                 ->getMock();
```

Comment:
It could be simplified to `$this->createMock(TranslatorInterface::class)`

Reply:
Done.